src/components/ToSTIX/Mapping.jsx

```diff
@@ -0,0 +1,356 @@
+import React from "react";
+import {
+  Add32,
+  Add20,
+  Close20,
+  Delete20,
+  SubtractAlt20,
+} from "@carbon/icons-react";
+import {
+  Button,
+  Dropdown,
+  MultiSelect,
+  TextInput,
+} from "carbon-components-react";
+import styles from "./to_stix.module.scss";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  addDataSourceField,
+  openNewObjectModal,
+  removeDataSourceField,
+  removeObject,
+  updateDataSourceField,
+  removeMappingField,
+  addMappingField,
+  updateMappingField,
+} from "../../store/actions/to_stix";
+
+const Mapping = () => {
+  const dispatch = useDispatch();
+  const mapping = useSelector((state) => state.toStix.mapping);
+
+  const getTransformers = () => {
+    return [
+      "StringToBool",
+      "EpochToTimestamp",
+      "FormatMac",
+      "FormatTCPProtocol",
+      "EpochSecondsToTimestamp",
+      "TimestampToMilliseconds",
+      "ToInteger",
+      "ToString",
+      "ToLowercaseArray",
+      "ToBase64",
+      "ToFilePath",
+      "ToDirectoryPath",
+      "ToFileName",
+      "ToDomainName",
+      "ToIPv4",
+      "DateTimeToUnixTimestamp",
+      "NaiveToUTC",
+      "TimestampToUTC",
+      "SetToOne",
+      "FilterIPv4List",
+      "FilterIPv6List",
+      "ValueToList",
+      "GraphIDToPID",
+    ];
+  };
+
+  const getObjects = () => {
+    return ["None", ...Object.keys(mapping)];
+  };
+
+  return (
+    <>
+      <div className="bx--row">
+        <div className="bx--col" style={{ textAlign: "right" }}>
+          <Button
+            renderIcon={Add32}
+            onClick={() => {
+              dispatch(openNewObjectModal());
+            }}
+          >
+            New Object
+          </Button>
+        </div>
+      </div>
+
+      <div className="bx--row">
+        <div className={`bx--col`}>
+          {Object.keys(mapping).length === 0 ? (
+            <p style={{ paddingTop: "1rem" }}>
+              There are currently no objects to show. Click the “New object”
+              button to start mapping or load configuration.
+            </p>
+          ) : (
+            Object.keys(mapping).map((objectName) => {
+              return (
+                <div
+                  key={objectName}
+                  className={`bx--row ${styles.object__item_box}`}
+                >
+                  <div className={`bx--col ${styles.object__item_content}`}>
+                    <div className={`bx--row`}>
+                      <div className={`bx--col ${styles.object_item__title}`}>
+                        "{objectName}"
+                      </div>
+                      <div className={`bx--col`} style={{ textAlign: "right" }}>
+                        <Add20
+                          className={`${styles.object_item__btn}`}
+                          style={{
+                            border: "1px solid #000000",
```

Comment:
can we move ` border: "1px solid #000000"` into `styles.object_item__btn`? I see most of the `styles.object_item__btn` uses the same border